src/engine/source/cmds/src/metrics.cpp

```diff
@@ -0,0 +1,181 @@
+{
+    using RequestType = eMetrics::Get_Request;
+    using ResponseType = eMetrics::Get_Response;
+    const std::string command {"metrics.manager/get"};
+
+    // Prepare the request
+    RequestType eRequest;
+    eRequest.set_instrumentname(instrumentName);
+    eRequest.set_scopename(scopeName);
+
+    // Call the API
+    const auto request = utils::apiAdapter::toWazuhRequest<RequestType>(command, details::ORIGIN_NAME, eRequest);
+    const auto response = client->send(request);
+    const auto eResponse = utils::apiAdapter::fromWazuhResponse<ResponseType>(response);
+
+    // Print value as json
+    const auto& value = eResponse.value();
+    const auto json = eMessage::eMessageToJson<google::protobuf::Value>(value);
+    std::cout << std::get<std::string>(json) << std::endl;
+}
+
+void runEnableInstrument(std::shared_ptr<apiclnt::Client> client, const std::string& scopeName, const std::string& instrumentName, bool status)
+{
+    using RequestType = eMetrics::Enable_Request;
+    using ResponseType = eMetrics::Enable_Response;
+    const std::string command {"metrics.manager/enable"};
+
+    RequestType eRequest;
+    eRequest.set_instrumentname(instrumentName);
+    eRequest.set_scopename(scopeName);
+    eRequest.set_status(status);
+
+    // Call the API
+    const auto request = utils::apiAdapter::toWazuhRequest<RequestType>(command, details::ORIGIN_NAME, eRequest);
+    const auto response = client->send(request);
+    const auto eResponse = utils::apiAdapter::fromWazuhResponse<ResponseType>(response);
+}
+
+void runListInstruments(std::shared_ptr<apiclnt::Client> client)
+{
+    using RequestType = eMetrics::List_Request;
+    using ResponseType = eMetrics::List_Response;
+    const std::string command {"metrics.manager/list"};
+
+    RequestType eRequest;
+
+    // Call the API
+    const auto request = utils::apiAdapter::toWazuhRequest<RequestType>(command, details::ORIGIN_NAME, eRequest);
+    const auto response = client->send(request);
+    const auto eResponse = utils::apiAdapter::fromWazuhResponse<ResponseType>(response);
+
+    // Print value as json
+    const auto& value = eResponse.value();
+    const auto json = eMessage::eMessageToJson<google::protobuf::Value>(value);
+
+    auto jsonStr = std::get<std::string>(json);
+    json::Json jsonObj(jsonStr.c_str());
+    auto jsonArray = jsonObj.getArray().value();
+    std::string result = "";
+    for (auto& element : jsonArray)
+    {
+        auto scopeValue = element.getString("/scope").value();
+        auto nameValue = element.getString("/name").value();
+        auto typeValue = element.getString("/type").value();
+        auto statusValue = element.getString("/status").value();
+        result += " " + scopeValue + "." + nameValue + "," + typeValue + "," + statusValue + "\n";
+    }
+
+    std::cout << result << std::endl;
+}
+
+void runTest(std::shared_ptr<apiclnt::Client> client)
```

Comment:
The idea of including the test metric was because we had not yet developed the other metrics and for an early testing stage it was useful. We can remove it in a future version if requested.